lua/lv-which-key/init.lua

```diff
@@ -179,17 +134,42 @@ local mappings = {
         s = {"<cmd>SessionSave<cr>", "Save Session"},
         l = {"<cmd>SessionLoad<cr>", "Load Session"}
     },
-    -- extras
-    z = {
-        name = "Zen",
-        s = {"<cmd>TZBottom<cr>", "toggle status line"},
-        t = {"<cmd>TZTop<cr>", "toggle tab bar"},
-        z = {"<cmd>TZAtaraxis<cr>", "toggle zen"}
-    }
+}
+
+local visualMappings = {
+    ["/"] = {":'<,'>CommentToggle<cr>", "Comment"},
```

Comment:
This is indeed the solution, replacing \<cmd> with :, but did you understand the rational ?
You only need to do ":CommentToggle<cr>"